content/_changelogs/9.6.0.md

```diff
@@ -0,0 +1,63 @@
+## 9.6.0
+
+_Released 4/25/2022_
+
+**Experiments:**
+
+- The Sessions beta feature has been enhanced by the new beta feature which
+  enables Multi-Origin support. To reflect the intent of using these experiments
+  together, the `experimentalSessionSupport` configuration option has been
+  replaced by the `experimentalSessionAndOrigin` configuration option. The
+  behaviors and usage of the `cy.session()` command has not been changed. Please
+  update your configuration to use the new `experimentalSessionAndOrigin` option
+  to continue to opt-in to this beta feature.
```

Comment:
This should just be in the features section, Experiments is not a normal section for changelogs. I really want the origin announcement to be at the top of the changelog, this is not as important to note.